src/otx/core/utils/tile_merge.py

```diff
@@ -448,3 +449,102 @@ def get_saliency_maps_from_masks(
 
         pred = {"labels": labels, "scores": scores, "masks": masks}
         return InstSegExplainAlgo.average_and_normalize(pred, num_classes)
+
+
+class SegmentationTileMerge(TileMerge):
```

Comment:
Could you add unit tests for this?

Reply:
It's covered. 
![image](https://github.com/user-attachments/assets/bdfe12b3-7d94-4808-a087-50c74ff3ffc7)